build/bin/network-validator.go

```diff
@@ -52,7 +60,7 @@ func main() {
 	config := reachabilityConfig{}
 	err := config.LoadFromYaml(*configFilePath)
 	if err != nil {
-		err = fmt.Errorf("unable to reach config file %v: %v", configFilePath, err)
+		err = fmt.Errorf("Unable to reach config file %v: %v", configFilePath, err)
```

Comment:
Minor, but errors is go start with lowercase, so this will throw a linting error

Reply:
yeah, although this is not recommended by linters, I wanted to ephemerally set this to catch the errors in https://github.com/openshift/osd-network-verifier/blob/main/pkg/cloudclient/aws/private.go#L284 or https://github.com/openshift/osd-network-verifier/blob/main/pkg/cloudclient/gcp/private.go#L237
but I can revert if this is an issue causing linters to be failing

Reply:
yeah, leaning on just `Unable to reach` is not a good way of generalizing the error catching mechanism, but we might refactor it in future PRs